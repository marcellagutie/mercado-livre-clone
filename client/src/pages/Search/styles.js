import styled from 'styled-components';

import { white } from '../../styles/colors';


export const Container = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
   flex-direction: column;
   background: ${white};
   border-radius: 4px;
   margin-bottom: 50px;

   hr {
      margin-bottom: 20px;
      width: 100%;
      height: 2px;
      border: 0;
      background: linear-gradient(to right, transparent, #ccc, transparent);
   }
`;

