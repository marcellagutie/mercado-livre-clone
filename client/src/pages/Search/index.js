import React from 'react';

import { Wrapper } from '../../styles/wrapper';
import Navigation from '../../components/Navigation';

export default function Search() {
   return (
      <Wrapper>
         <Navigation categories={['Home']} />
      </Wrapper>
   );
}
