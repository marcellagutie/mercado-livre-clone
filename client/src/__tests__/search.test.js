import React from 'react';
import renderer from 'react-test-renderer';

import Search from '../pages/Search';

it('renders correctly the empty page', () => {
  const tree = renderer.create(<Search />).toJSON();
  expect(tree).toMatchSnapshot();
});