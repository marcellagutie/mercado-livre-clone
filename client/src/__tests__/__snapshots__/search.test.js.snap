// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the empty page 1`] = `
<div
  className="sc-bcXHqe yTffH"
>
  <div
    className="sc-gswNZR eYnnWB"
  >
    <span>
      <a
        href="/"
      />
    </span>
  </div>
</div>
`;
